examples/eth1_add_route_rule.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Adding only one rule seems to be an artificial example or does this have a real use case? Can we show a real use case for the rules instead?

Reply:
Real use case is https://www.ovirt.org/develop/release-management/features/network/multiple-gateways.html

which require two interface and each just have 1 direct route and 1 default gateway.
I will use that one to create a the example yaml.